cdap-hbase-compat-1.0-cdh/src/main/java/co/cask/cdap/data2/util/hbase/DefaultHBase10CDHDDLExecutor.java

```diff
@@ -33,4 +48,28 @@ public HTableDescriptor getHTableDescriptor(TableDescriptor descriptor) {
   public TableDescriptor getTableDescriptor(HTableDescriptor descriptor) {
     return HBase10CDHTableDescriptorUtil.getTableDescriptor(descriptor);
   }
+
+  @Override
```

Comment:
Note this uses ConnectionFactory, which is not available in HBase before 1.0. Because the hbase-compat-base depends on HBase 0.98, this cannot be implemented in the base class and must unfortunately be copied to all compat modules. I tried to change the base to depend on HBase 1.0, but that would require updating our Hadoop dependency to 2.5.1+, and it appears to big of a change so short before code freeze.